crossdock/docker-compose.yml

```diff
@@ -82,6 +82,8 @@ services:
         image: jaegertracing/test-driver
         ports:
             - "8080"
+        environment:
+            - JAEGER_COLLECTOR_HC_HOST_PORT=jaeger-collector:${JAEGER_COLLECTOR_HC_PORT}
```

Comment:
test-driver is unchanged in this PR, what effect does giving it this new env var has?

Reply:
This env var is consumed by `test_driver`. 

OTEL collector used a different health check port than Jaeger collector hence we have to specify it here.